src/System.Net.Primitives/src/Resources/Strings.resx

```diff
@@ -117,2383 +117,55 @@
-  </data>
-  <data name="net_uri_PortOutOfRange" xml:space="preserve">
-    <value>A derived type '{0}' has reported an invalid value for the Uri port '{1}'.</value>
-  </data>
-  <data name="net_uri_UserDrivenParsing" xml:space="preserve">
-    <value>A derived type '{0}' is responsible for parsing this Uri instance. The base implementation must not be used.</value>
-  </data>
-  <data name="net_uri_AlreadyRegistered" xml:space="preserve">
-    <value>A URI scheme name '{0}' already has a registered custom parser.</value>
-  </data>
-  <data name="net_uri_NeedFreshParser" xml:space="preserve">
-    <value> The URI parser instance passed into 'uriParser' parameter is already registered with the scheme name '{0}'.</value>
-  </data>
-  <data name="net_uri_CannotCreateRelative" xml:space="preserve">
-    <value>A relative URI cannot be created because the 'uriString' parameter represents an absolute URI.</value>
-  </data>
-  <data name="net_uri_InvalidUriKind" xml:space="preserve">
-    <value>The value '{0}' passed for the UriKind parameter is invalid.</value>
-  </data>
-  <data name="net_uri_BadUnicodeHostForIdn" xml:space="preserve">
-    <value>An invalid Unicode character by IDN standards was specified in the host.</value>
-  </data>
-  <data name="net_uri_GenericAuthorityNotDnsSafe" xml:space="preserve">
-    <value>The generic authority '{0}' is not a valid dns name.</value>
-  </data>
-  <data name="net_uri_NotJustSerialization" xml:space="preserve">
-    <value>UriComponents.SerializationInfoString must not be combined with other UriComponents.</value>
-  </data>
   <data name="net_emptystringcall" xml:space="preserve">
     <value>The parameter '{0}' cannot be an empty string.</value>
   </data>
-  <data name="net_emptystringset" xml:space="preserve">
-    <value>This property cannot be set to an empty string.</value>
-  </data>
-  <data name="net_headers_req" xml:space="preserve">
-    <value>This collection holds response headers and cannot contain the specified request header.</value>
-  </data>
-  <data name="net_headers_rsp" xml:space="preserve">
-    <value>This collection holds request headers and cannot contain the specified response header.</value>
-  </data>
-  <data name="net_headers_toolong" xml:space="preserve">
-    <value>Header values cannot be longer than {0} characters.</value>
-  </data>
-  <data name="net_WebHeaderInvalidControlChars" xml:space="preserve">
-    <value>Specified value has invalid Control characters.</value>
-  </data>
-  <data name="net_WebHeaderInvalidCRLFChars" xml:space="preserve">
-    <value>Specified value has invalid CRLF characters.</value>
+  <data name="dns_bad_ip_address" xml:space="preserve">
+    <value>An invalid IP address was specified.</value>
   </data>
-  <data name="net_WebHeaderInvalidHeaderChars" xml:space="preserve">
-    <value>Specified value has invalid HTTP Header characters.</value>
+  <data name="net_container_add_cookie" xml:space="preserve">
```

Comment:
I assume these "adds" are just an artifact of the diff, and you're really only just deleting stuff?